meson.build

```diff
@@ -0,0 +1,17 @@
+project(
+  'flatpak-xdg-utils', 'c',
+  version: '0.0.1',
+  license: 'LGPL-2.1+',
+)
+
+conf = configuration_data()
+conf.set_quoted('GETTEXT_PACKAGE', meson.project_name())
+conf.set_quoted('PACKAGE_VERSION', meson.project_version())
+config_h = configure_file(output: 'config.h', configuration: conf)
+
+add_global_arguments('-D_GNU_SOURCE=1', language: 'c')
+
+gio = dependency('gio-2.0', version: '>= 2.50')
+gio_unix = dependency('gio-unix-2.0')
```

Comment:
This is redundant anyway. `gio-unix` always pulls in `gio` so you can just drop it.

Reply:
Thanks, fixed in the latest push to this branch.